instrumentation/apache-httpasyncclient-4.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachehttpasyncclient/ApacheHttpClientRequest.java

```diff
@@ -143,4 +147,13 @@ private static URI getCalculatedUri(HttpHost httpHost, URI uri) {
       return null;
     }
   }
+
+  @Nullable
+  public InetSocketAddress peerAddress() {
```

Comment:
`getPeerAddress` to conform? (or maybe `getPeerSocketAddress` if going with other rename thought)